apps/web-staking/src/app/hooks/useGetMaxKeyPerPool.ts

```diff
@@ -6,7 +6,7 @@ import { useEffect, useState } from "react";
 import { useAccount } from "wagmi";
 
 export const useGetMaxKeyPerPool = () => {
-  const [maxKeyPerPool, setMaxKeyPerPool] = useState(750);
+  const [maxKeyPerPool, setMaxKeyPerPool] = useState(1000);
```

Comment:
This will not help us if the values change again, we would have to update again and again.

We need to follow the pattern where, if its loading nothing is showed and once the data is fetched from the blockchain we want to update the value,.

Make sure to not stop this hook if the wallet is not connected, thats the actual issue. We don't need a wallet connection for blockchain reads, so this should not require the chainId to be set, rather it should just fetch from the default network if no wallet is connected

Reply:
Updated hook, now it will not stop if the wallet is not connected, if the wallet not connected the maximum value of keys will come from the default network